hugegraph-llm/src/hugegraph_llm/resources/demo/css.py

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+CSS = r"""
```

Comment:
How about using native `.css` instead of `css.py`?

Reply:
> How about using native `.css` instead of `css.py`?

Both fine for me (there is not much CSS now, but when multiple places may be embedded, using `.py` may be more convenient for using. we could separate it when there is a global css need/modification)